MFVNeutralino/plugins/TrackMover.cc

```diff
@@ -121,7 +134,20 @@ void MFVTrackMover::produce(edm::Event& event, const edm::EventSetup&) {
     for (const pat::Jet& jet : *jets) {
       if (jet.pt() < min_jet_pt || track_ref_getter.tracks(event, jet).size() < min_jet_ntracks)
         continue;
-
+      int muonjets = 0;
+      for (const pat::Muon& muon : *muons) {
+        bool isLooseMuon = muon.passed(reco::Muon::CutBasedIdLoose);
```

Comment:
Is this the ID we'll use elsewhere in the analysis? I would have expected this muon and the electron below it to have dxy/dz cuts (since this is to select prompt leptons in WH/ZH, I think), and I would have expected to see lepton isolation applied to the muon (I think it is part of the electron ID you're using, but be sure to check the e/gamma Twikis).